connector/avro/src/test/scala/org/apache/spark/sql/avro/AvroSuite.scala

```diff
@@ -946,6 +946,35 @@ abstract class AvroSuite
     }
   }
 
+  Seq(
+    "time-millis",
```

Comment:
@zeruibao can you show me where is the `time-millis` and `time-micros` handled in the deserializer?

Reply:
When the logical type is `time-millis` and `time-micros` , `SchemaConverters.toSqlType(avroType).dataType` is also longType. So, it's handled.